crates/wasmparser/src/binary_reader.rs

```diff
@@ -105,10 +166,12 @@ impl<'a> BinaryReader<'a> {
         }
     }
 
+    /// Gets the original position of the binary reader.
```

Comment:
FWIW, I think using `deny(missing_docs)` is a net positive, and we can always `allow(missing_docs)` in certain places if it makes sense. But it is a much better default, IMO.

Reply:
Happy to remove any redundant documentation in favor of targeted `allow(missing_docs)`, but I definitely prefer denying it at the crate level as it really makes it clear when a type gets accidentally leaked from the crate unintentionally (I rarely add doc comments to internal types), especially since we have some `pub use self::x::*` in a few places for simplicity.